app/src/androidTest/java/ch/epfl/sdp/drone3d/ui/map/MissionInProgressActivityTest.kt

```diff
@@ -1,36 +1,68 @@
-/*
- * Copyright (C) 2021  Drone3D-Team
- * The license can be found in LICENSE at root of the repository
- */
-
```

Comment:
I think we agreed the display the header in every file.

Reply:
It was removed for some reason